templates/admin/workers/index.html.ep

```diff
@@ -45,6 +45,7 @@
                     <th>Class</th>
                     <th>Arch</th>
                     <th>Status</th>
+                    <th>Version</th>
```

Comment:
Just to be clear here, it would be os-autoinst version + api interface version? or only the interface version?... If it's the latter, we could benefit greatly from https://github.com/os-autoinst/os-autoinst/pull/904 althought that might be a bit out of scope here, but would be a nice follow up... so that the worker page becomes a bit more useful.

Reply:
@foursixnine 
To avoid confusions, there are three versions
- Worker version
- openQA version
- os-autoinst version

Explanation of the versions:
1. Worker version - this is the version of the commands that the worker uses to communicate via websocket with the openqa. This value must be equal with openqa version, otherwise the worker will shutdown
2. openqa version - this is the version of the commands that the worker uses to communicate via websocket with the worker. This value must be equal with worker version, otherwise the worker will shutdown
3. os-autoinst version - this is the version of the autoinst that the test developer need to follow in order the tests to run correctly on that worker.

This was developed as separated developments, there was no os-autoinst version when this was developed. 

Will "Websocket api version" be a better name than "version"?
Let me check the possibility of adding the os-autoinst version.

Reply:
> os autoinst version has been there all along 
yeah true XD
I was referring to the PR you linked.

Changed the names and added the os-autoinst version :-)